client/operation_read_object_group.go

```diff
@@ -141,6 +185,23 @@ func mapBucketTaggingToResponse(tagging *s3.GetBucketTaggingOutput, v *ReadObjec
 		return err
 	}
 
+	// QUICKFIX
+	// https://klarna.atlassian.net/browse/OX-1168
+	// until ChaosSearch fixes this, this workaround converts back the filter_json into the original format!
+	// For example:
+	//     `{"AND":[{"field":"key","regex":{"pattern":".*","strict":true}}]}`
+	// is converted back into
+	//     `{"AND":[{"field":"key","regex":".*"}]}`
+	needle := "pattern"
+	inputFilterJSON := v.FilterJSON
+	if strings.Contains(inputFilterJSON, needle) {
```

Comment:
this is a bit too risky IMO. If I understand ChaosSearch's reply - that `".*"` is equivalent to `{"pattern":".*","strict":true}}` in the `regex` attribute - let's check only that. Also, let's add a comment to explain why we're doing it (linking to Klarna's internal ticket doesn't help since this is a public repo), which is that this is to work around a non-versioned breaking API change in ChaosSearch.